corehq/apps/reports/templates/reports/standard/base_template.html

```diff
@@ -82,7 +82,7 @@
          data-target="#create-new-report-left-nav"
          data-slug="{{ "REPORT_BUILDER_V2"|ui_notify_slug }}"
          role="alert">
-      {% include 'notifications/partials/report_builder_v2_oct2017.html' %}
+      {% include 'notifications/partials/bootstrap3/report_builder_v2_oct2017.html' %}
```

Comment:
This whole notification could just be deleted, right? We might as well delete all the old ones.

Reply:
yes, i think fine to do in a followup PR